app/assets/stylesheets/partials/_admin_workshop.scss

```diff
@@ -2,3 +2,10 @@
   padding-top: 10px;
 }
 
+#workshop_description {
+  height: 150px;
+}
+
+.lead {
```

Comment:
@turger was wondering if there's a Foundation class we could use here instead of defining our own.